Field-Robotics-Lab/uuv_manipulators
# Oberon7 MoveIt Configuration and Gazebo Controllers

Required by field-robotics-lab/dave#206
Requires field-robotics-lab/dockwater#21

### Summary
Rather than change existing files in `oberon7_description`, new ones were created to faciliate configuration via MoveIt.

### To test
See the "two arms in empty world" test  in field-robotics-lab/dave#206. The Gazebo launch file is the only file called outside of `uuv_manipulators`.

### Highlights `oberon7_description/urdf`
`oberon7.urdf.xacro`
Redefines oberon7, allows for namespace changes for multiple arms and an argument for `transmission_hw_interface`. Moves gazebo and transmission definitions to separate files, `oberon7.transmission.xacro`, `oberon7.gazebo.xacro`, and `common.gazebo.xacro`
